src/js/views/patients/schedule/schedule_views.js

```diff
@@ -230,16 +251,62 @@ const ScheduleListView = CollectionView.extend({
       state: this.state,
     };
   },
-  emptyView: EmptyView,
+  emptyView() {
+    if (this.state.get('searchQuery')) {
+      return EmptyFindInListView;
+    }
+
+    return EmptyView;
+  },
   viewComparator(viewA, viewB) {
     return alphaSort('asc', viewA.model.get('date'), viewB.model.get('date'));
   },
   initialize({ state }) {
     this.state = state;
 
-    this.listenTo(state, {
-      'select:all': this.render,
-      'select:none': this.render,
+    this.listenTo(state, 'change:searchQuery', this.searchList);
+  },
+  searchList(state, searchQuery) {
+    // Must remove the parent view's filter so we have access to previously filtered children
```

Comment:
I think we should pass the state down to the children so they can watch for the searchQuery change so each day works just like the regular worklist / matcher / setFilter
In this collectionview I think we should always use
```js
viewFilter(view) {
  return !view.isEmpty();
}
```
Then we can do `childViewEvents` in this view  with `'filter': `onChildFilter` where `onChildFilter` calls `this.filter()`.  Then in the `initialize` we should debounce the `onChildFilter` to something like 60.
Then what's in `setVisibleChildren` should be in an event like `onRenderChildren`

This way if things are added/removed from these collections for other reasons they'll follow the same logic.